english/work-out/work-out.md

```diff
@@ -0,0 +1,8 @@
+Now that you've made a good life desicion and decided to work out, you now must
+choose your method. Do you:
+
+Work that cardiovascular system by going for a [run](/run/run.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.